packages/class/src/config.json

```diff
@@ -0,0 +1,121 @@
+{
+  "type": "object",
+  "properties": {
+    "initialState": {
+      "type": "object",
+      "properties": {
+        "h_0": {
+          "type": "number",
+          "default": 200,
+          "title": "ABL height [m]"
+        },
+        "theta_0": {
+          "type": "number",
+          "default": 288,
+          "title": "Mixed-layer potential temperature [K]",
```

Comment:
Would be nice to have units as a separate attribute

Reply:
Created #52 for it